back/spec/acceptance/projects_spec.rb

```diff
@@ -635,6 +635,26 @@
         expect(copied_project.title_multiloc['en']).to include(source_project.title_multiloc['en'])
       end
 
+      example 'Copy non-draft project', document: false do
+        expect(source_project.admin_publication.publication_status).not_to eq 'draft'
+
+        do_request
+        assert_status 201
+
+        copied_project = Project.find(json_response.dig(:data, :id))
+        expect(copied_project.admin_publication.publication_status).to eq 'draft'
+      end
+      
+      example 'Copy a project in a folder', document: false do
+        folder = create(:project_folder, projects: [source_project])
+
+        do_request
+        assert_status 201
+
+        copied_project = Project.find(json_response.dig(:data, :id))
+        expect(copied_project.folder_id).to eq folder.id
+      end
```

Comment:
I'm not sure anymore what happens if a project moderator copies a project in a folder. In principle, they should not be allowed to create projects unless they also manage the folder.

Reply:
Very good point!

Looking into it with a quick acceptance test, it seems the project is copied, but not into the folder… which is less than ideal, though it makes logical sense.

Reply:
Actually, it makes perfect sense, and it also still allows folder mods to copy into source project folder, so all good. i'll add tests to document all this.